src/main/kotlin/com/exactpro/th2/ws/client/api/impl/WebSocketClient.kt

```diff
@@ -238,30 +241,30 @@ class WebSocketClient(
         return null
     }
 
-    private class WebSocketClientSettings(private val builder: WebSocket.Builder, private val uriBuilder: URIBuilder) : IClientSettings {
+    private class WebSocketClientSettings(private val builder: WebSocket.Builder, private val baseUri: URI) : IClientSettings {
+        private var uriBuilder: URIBuilder? = null
+        val uri: URI get() = uriBuilder?.build() ?: baseUri
```

Comment:
I thought about it again and maybe we can always create builder to avoid fiddling with nulls and nullable-types (I haven't figured out how to make it look better)